src/mapped-types.ts

```diff
@@ -5,14 +5,14 @@
 
 /**
  * Primitive
- * @desc Type representing primitive types in TypeScript: `number | boolean | string | symbol`
+ * @desc Type representing primitive types in TypeScript: `number | bigint | boolean | string | symbol`
  * @example
  *   type Various = number | boolean | string | symbol | object;
```

Comment:
Could you please update example as well @9SMTM6 ?

Reply:
If you wish so. My reason for not changing it:

The example is still correct as it is currently, you just exclude a type that was never there. Maybe it causes an syntax error, I'm not that familiar with your `Exclude` type, but I wouldn't think so.

And if it works like that - as I'd strongly suspect - the way it is it demonstrates that you don't in fact need to be careful with excluding primitives from any type (that the type from which to exclude doesn't have to be either a complete superset of primitive, but can have any subset, including empty, of primitive).

In fact, I'd maybe even suggest stripping some more primitive types from the example.

Reply:
Ok, I like your updated suggestion, go for it! Thanks